src/components/TypeComponents/SimpleSearch/Widget/getDefaultRenderFunction.js

```diff
@@ -25,15 +25,13 @@ const getDefaultRenderFunction = ({ accessPath, arrayDisplayPath, name, valueTyp
       case 'date': {
         return (data) => {
           const date = get(data, accessPath);
```

Comment:
While we are here, can we move all the `const date = get(data, accessPath);` to the top of the if condition above the switch statement and reuse the constant. Also, lets use optional chaining instead of the lodash get throughout this file.

Reply:
Hm, but this const is not always named the same. It's depending on the valueType. Should we then find a general name for that constant?

Reply:
I see. So, lets just take out the constant and instead have `data?.accessPath && <FormattedUTCDate value={date} />`. Yeah, its not part of the issue, but its good to clean it up when we are touching these files.

Reply:
And when I use optional chaining I'm getting this error 

Error: Error: Component(...): Nothing was returned from render. This usually means a return statement is missing. Or, to render nothing, return null.
at reconcileChildFibers (http://localhost:3000/bundle.js:395104:23)
at reconcileChildren (http://localhost:3000/bundle.js:397518:28)
at mountIndeterminateComponent (http://localhost:3000/bundle.js:398298:5)
at beginWork (http://localhost:3000/bundle.js:399352:16)
at HTMLUnknownElement.callCallback (http://localhost:3000/bundle.js:380944:14)
at Object.invokeGuardedCallbackDev (http://localhost:3000/bundle.js:380993:16)
at invokeGuardedCallback (http://localhost:3000/bundle.js:381048:31)
at beginWork$1 (http://localhost:3000/bundle.js:403959:7)
at performUnitOfWork (http://localhost:3000/bundle.js:402910:12)
at workLoopSync (http://localhost:3000/bundle.js:402886:22)

in Unknown (created by ResizedTable)
in td (created by ResizedTable)
in tr (created by ResizedTable)
in tbody (created by ResizedTable)
in table (created by ResizedTable)
in div (created by ResizedTable)
in ResizedTable (created by SimpleTable)
in SimpleTable (created by SimpleSearch)
in SimpleSearch (created by RenderWidget)
in div (created by Widget)
in div (created by Widget)
in div (created by Widget)
in FocusTrap (created by HotKeys)
in HotKeys (created by Context.Consumer)
in HotKeys-HotKeys (created by HasCommand)
in HasCommand (created by Widget)
in Widget (created by RenderWidget)
in RenderWidget (created by Dashboard)
in div (created by Dashboard)
in div (created by Dashboard)
in div (created by Dashboard)
in Dashboard (created by DashboardRoute)
in DashboardRoute (created by Context.Consumer)
in Route (created by Route)
in Route (created by App)
in Switch (created by App)
in FocusTrap (created by HotKeys)
in HotKeys (created by Context.Consumer)
in HotKeys-HotKeys (created by HasCommand)
in HasCommand (created by App)
in FocusTrap (created by HotKeys)
in HotKeys (created by Context.Consumer)
in HotKeys-HotKeys (created by CommandList)
in CommandList (created by App)
in App (created by Context.Consumer)
in Titled (created by Context.Consumer)
in GetContext (created by TitleManager)
in TitleManager (created by Context.Consumer)
in WithStripes(TitleManager) (created by Context.Consumer)
in ErrorBoundary (created by RouteErrorBoundary)
in RouteErrorBoundary (created by Context.Consumer)
in div (created by Context.Consumer)
in ModuleHierarchyProvider (created by Context.Consumer)
in AddContext (created by Context.Consumer)
in Route (created by Context.Consumer)
in Switch (created by RootWithIntl)
in main (created by ModuleContainer)
in ModuleContainer (created by RootWithIntl)
in AppCtxMenuProvider (created by RootWithIntl)
in div (created by MainContainer)
in MainContainer (created by LastVisited)
in LastVisited (created by Context.Consumer)
in WithModules(LastVisited) (created by Context.Consumer)
in withRouter(WithModules(LastVisited)) (created by RootWithIntl)
in Router (created by RootWithIntl)
in Provider (created by RootWithIntl)
in FocusTrap (created by HotKeys)
in HotKeys (created by Context.Consumer)
in HotKeys-HotKeys (created by RootWithIntl)
in Titled (created by Context.Consumer)
in GetContext (created by TitleManager)
in TitleManager (created by Context.Consumer)
in WithStripes(TitleManager) (created by RootWithIntl)
in ModuleTranslator (created by Context.Consumer)
in injectIntl(ModuleTranslator) (created by RootWithIntl)
in RootWithIntl (created by Root)
in IntlProvider (created by Root)
in QueryClientProvider (created by Root)
in ApolloProvider (created by Root)
in ErrorBoundary (created by Root)
in Root (created by Context.Consumer)
in WithModules(Root) (created by Connect(WithModules(Root)))
in Connect(WithModules(Root)) (created by HotExportedConnect(WithModules(Root)))
in AppContainer (created by HotExportedConnect(WithModules(Root)))
in HotExportedConnect(WithModules(Root)) (created by StripesCore)
in StripesCore

Reply:
Did you add the return statement?

Reply:
> I see. So, lets just take out the constant and instead have `data?.accessPath && <FormattedUTCDate value={date} />`. Yeah, its not part of the issue, but its good to clean it up when we are touching these files.

That's not working, first it then has to be `data?.accessPath && <FormattedUTCDate value={data.accessPath} />`
and as I wrote, optional chaining is giving me the error above. I left all as it was and only tried to change 
`const date = get(data, accessPath);`
 `return date && <FormattedUTCDate value={date} />;` 
to
`return data?.accessPath && <FormattedUTCDate value={data.accessPath} />;`